README.md

```diff
@@ -2,16 +2,16 @@
 
 [![Build Status](https://travis-ci.org/igneus/church-calendar-api.svg?branch=master)](https://travis-ci.org/igneus/church-calendar-api)
 
-Browse Roman Catholic liturgical calendar through a web interface
-or obtain it's data in JSON format through an API.
+Browse the Roman Catholic liturgical calendar through a web interface
+or obtain its data in JSON format through an API.
```

Comment:
It's is a contraction of the two words 'it is'. Thus the original could be reread as 'or obtain it is data in JSON format through an API'.

Its on the other hand suggests possession/ownership.

Reply:
This is something [I've been writing the wrong way](https://www.lawlessenglish.com/english-mistakes/its/) my whole life. Thanks.